demos/step.ts

```diff
@@ -40,7 +42,7 @@ console.log(state.vector.toString())
 // console.log(state.aggregatePolarization())
 
 console.log('Add:')
-state.addPhotonIndicator(sizeX - 1, sizeY - 1, '^', 'H')
+state.addPhotonIndicator({ x: sizeX - 1, y: sizeY - 1, direction: 0, h: Cx(0, 1), v: Cx(1, 0) })
```

Comment:
This think should be reverted.

First, `fromIndicator` everything else is a specific case when we use a basis state (NOT any amplitude). Vide in `Vector` and `Operator`.

Second, in general it is a questionable pattern that has a function with a single argument, wrapping all subargument.